fs/accounting/stats.go

```diff
@@ -37,12 +39,130 @@ type StatsInfo struct {
 
 // NewStats cretates an initialised StatsInfo
 func NewStats() *StatsInfo {
-	return &StatsInfo{
+	s := &StatsInfo{
 		checking:     newStringSet(fs.Config.Checkers),
 		transferring: newStringSet(fs.Config.Transfers),
 		start:        time.Now(),
 		inProgress:   newInProgress(),
 	}
+
+	rc.Add(rc.Call{
+		Path:  "stats/transfers",
+		Fn:    s.bwStats,
+		Title: "Get transfer stats",
+		Help: `
+Show statistics for all transfers.
+`,
+	})
+
+	rc.Add(rc.Call{
+		Path:  "stats/reset",
+		Fn:    s.bwResetStats,
+		Title: "Reset transfer stats",
+		Help: `
+Resets the global transfer statistic counters.
+`,
+	})
+
+	return s
+}
+
+// reset statistics as requested by rc stats/reset
+func (s *StatsInfo) bwResetStats(in rc.Params) (out rc.Params, err error) {
+	out = make(rc.Params)
+
+	s.mu.Lock()
+	s.bytes = 0
+	s.errors = 0
```

Comment:
Note that resetting errors to 0 potentially changes the exit code which may/may not be desirable!  It is OK if we document it I guess..